src/submission/exceptions/exceptions.ts

```diff
@@ -129,7 +129,8 @@ const isSingleString = (value: DeepReadonly<dictionaryEntities.SchemaTypes>): va
  */
 const isAllowedTypeForException = (
   value: DeepReadonly<dictionaryEntities.SchemaTypes>,
-): value is string | [string] => typeof value === 'string' || isSingleString(value);
+): value is string | [string] | undefined =>
+  typeof value === 'string' || isSingleString(value) || typeof value === 'undefined';
```

Comment:
As discussed offline, the allowed types for numeric and string fields are different. String fields are only allowed to be `string` or `[string]`. With numeric fields, we accept a number that passes validation or `undefined` with the exception, so we'd be looking for:

```ts
const isAllowedTypeForNumberFieldException = (
  value: DeepReadonly<dictionaryEntities.SchemaTypes>,
): undefined => value === undefined;
```

or similar, if needed.